src/Microsoft.AspNetCore.Blazor.Razor.Extensions/ComponentDocumentClassifierPass.cs

```diff
@@ -11,32 +11,48 @@
 
 namespace Microsoft.AspNetCore.Blazor.Razor
 {
+    /// <summary>
+    /// A <see cref="DocumentClassifierPassBase"/> that recognizes Blazor components.
+    /// </summary>
     public class ComponentDocumentClassifierPass : DocumentClassifierPassBase, IRazorDocumentClassifierPass
```

Comment:
This should be internal instead, there's not scenario for user code to interact with this